src/Bundle/spec/Renderer/Adapter/EmailTwigAdapterSpec.php

```diff
@@ -20,7 +20,7 @@
 use Sylius\Component\Mailer\Renderer\Adapter\AbstractAdapter;
 use Sylius\Component\Mailer\Renderer\RenderedEmail;
 use Sylius\Component\Mailer\SyliusMailerEvents;
-use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
```

Comment:
Shouldn't we create a polifil for this behavior?

Reply:
/cc @pamil

Reply:
Added for Event class